src/js/components/Table/doc.js

```diff
@@ -6,7 +6,7 @@ import { themeDocUtils } from '../../utils/themeDocUtils';
 
 export const doc = Table => {
   const DocumentedTable = describe(Table)
-    .availableAt(getAvailableAtBadge('Table'))
+    .availableAt(getAvailableAtBadge('Visualizations-Table'))
```

Comment:
Could this change be a problem for the codesandbox URL that is also placed on `availableAt`?

Reply:
Yes this will cause a problem with the codesandbox URL, I'm looking into finding a solution

Reply:
Proposed fix here https://github.com/grommet/grommet/pull/4749